### ISSUE TITLE
Rapid Repeated "Calendar Username is incorrect" errors prevent you from fixing them

### ISSUE THREAD

**Issue by nonplayer** [pbek/QOwnNotes#1872]
Issue: if the calendar option is selected and the username is incorrect, QOwnNotes repeatedly informs you with messages that steal focus and prevent interaction with other menus.

#### Expected behaviour

Clicking "OK" should resolve the error for more than a couple of seconds. The appearance of the error dialog should not prevent interaction with the screens where it can be resolved.

#### Actual behaviour

Clicking "OK" just delays the error, which repeats again a couple of seconds later. Each time it reappears, you are locked out of all aspects of the app until you click OK.

Using version 20.9.5 on Win10x64

**Comment by nonplayer** [comment]
I eventually corrected the error cause, but the fact that it repeated so quickly seems bug-worthy.

**Comment by pbek** [comment]
Hm, I cannot reproduce that. I only get a message once.

![image](https://user-images.githubusercontent.com/1798101/93422541-409c2c80-f8b4-11ea-95e9-ddd214b7cd3d.png)

**Comment by pbek** [comment]
Can you still reproduce the dialog and provide that settings you have set (see issue template)?

**Comment by nonplayer** [comment]
Now that I finally fixed it, I can't replicate. But here's what I did:

- I created a new local Nextcloud instance and moved all my online notes to the new one
- in QON I changed the note folder to the new synched note folder from the new server.
- In QON I changed the default NextCloud server data to the new data. Then clicked OK.

After a short bit I started getting flooded with Calendar User is Incorrect messages, the only button was "OK" and there was no "Open Settings" button. While manually trying to find the calendar setting, the messages kept repeating. After unchecking the to-do list option and clicking OK out of the settings, the messages stopped.

Oddly, after going back in and re-checking the to-do list, the integration worked again, and no more messages came.
